internal/pkg/template/upgrade/context.go

```diff
@@ -25,7 +25,7 @@ func NewContext(ctx context.Context, templateRef model.TemplateRef, objectsRoot
 	}
 
 	// Register existing IDs, so they will be reused
-	configs := search.ConfigsForTemplateInstance(projectState.RemoteObjects().ConfigsWithRowsFrom(targetBranch), instanceId)
+	configs := search.ConfigsForTemplateInstance(projectState.LocalObjects().ConfigsWithRowsFrom(targetBranch), instanceId)
```

Comment:
Na viacerych miestach som mal chybne `Remote` namiesto `Local` (refactoring zabrani takymto problemom).
CLI aj API pracuje s local state, takze takto je to spravne.